lib/restAPIs/userManagement.js

```diff
@@ -19,49 +26,87 @@ module.exports = class RestTopic extends RestUtil {
     if (abbreviationBegins) {
       queryParameter.abbreviationBegins = abbreviationBegins;
     }
-    return this._makeRequest(this._getBase(), queryParameter, 'GET')
-      .then(result => this.throwIfFailed(result, 200));
+    return this._makeRequest(
+      this._getBase(),
+      queryParameter,
+      'GET'
+    ).then(result => this.throwIfFailed(result, 200));
   }
 
-  getUser(userId, fields){
+  /**
+   * @public Get info for a specific user
+   * @param {String} userId - User ID
+   * @param {String} fields - Top level fields to be included in the response
+   */
+  getUser(userId, fields) {
     assert(userId, 'No userId provided');
-    
+
     const queryParameter = buildCommonQuery(fields);
-    return this._makeRequest(`${this._getBase()}/${userId}`, queryParameter, 'GET')
-      .then(result => this.throwIfFailed(result, 200));
+    return this._makeRequest(
+      `${this._getBase()}/${userId}`,
+      queryParameter,
+      'GET'
+    ).then(result => this.throwIfFailed(result, 200));
   }
-
-  createUser(newUserBody = {}, fields){
+  /**
+   * @public Create a new user
+   * @param { Object} newUserBody - User information
+   * @param { String } fields - Top level fields to be included in the response
```

Comment:
Spacing `{ String }` -> `{String}` and primitive types `String` -> `string`

Reply:
Done.